WalletWasabi.Tests/UnitTests/WabiSabi/Integration/WabiSabiHttpApiIntegrationTests.cs

```diff
@@ -134,8 +137,8 @@ public async Task SoloCoinJoinTestAsync(long[] amounts)
 		}
 
 		[Theory]
-		[InlineData(new long[] { 20_000_000, 40_000_000, 60_000_000, 80_000_000 })]
-		[InlineData(new long[] { 10_000_000, 20_000_000, 30_000_000, 40_000_000, 100_000_000 })]
+		// [InlineData(new long[] { 20_000_000, 40_000_000, 60_000_000, 80_000_000 })]
```

Comment:
Is this an intentional change?

Reply:
Thanks: https://github.com/zkSNACKs/WalletWasabi/pull/6408